docs/en/sql-reference/statements/select/intersect.md

```diff
@@ -29,7 +29,7 @@ The condition could be any expression based on your requirements.
 Here is a simple example that intersects the numbers 1 to 10 with the numbers 3 to 8:
 
 ```sql
-SELECT number FROM numbers(1,10) INTERSECT SELECT number FROM numbers(3,6);
+SELECT number FROM numbers(1,10) INTERSECT SELECT number FROM numbers(3,8);
```

Comment:
According to the [docs](https://clickhouse.com/docs/en/sql-reference/table-functions/numbers), this code was right and returned the correct result. You can check it here: https://fiddle.clickhouse.com/746679d8-6646-470e-9646-0afaceeca1c8